File: uri/produto/produto.go
package main

import (
	"C"
	"fmt"
)

func produto(a int32, b int32) int32 {
	return (a * b)
}

func main() {
	var a, b int32
	fmt.Scanf("%d\n", &a)
	fmt.Scanf("%d\n", &b)
	fmt.Printf("PROD = %d\n", produto(a, b))
}

File: uri/produto/produto.c
#include <stdio.h>

int produto( int a, int b)
{
	return a * b;
}

int main( int argc, char *argv[] )
{
	int a, b;
	scanf("%d", &a);
	scanf("%d", &b);
	printf("PROD = %d\n", produto(a, b));
	return 0;
}
